doc/version_history.rst

```diff
@@ -0,0 +1,28 @@
+.. py:currentmodule:: lsst.ts.opensplice_rpm
+
+.. _lsst.ts.opensplice_rpm.version_history:
+
+###############
+Version History
+###############
+
+v-15
```

Comment:
How about we use "build" instead of version? since this is the nomenclature on rpms.

build 15 or b15?